integration/env/azure.go

```diff
@@ -26,6 +26,8 @@ const (
 	EnvVarBastionPublicSSHKey       = "BASTION_PUBLIC_SSH_KEY"
 
 	EnvVarCircleBuildNumber = "CIRCLE_BUILD_NUM"
+
+	EnvVarLatestOperatorRelease = "LATEST_OPERATOR_RELEASE"
```

Comment:
Instead of calling our catalog to figure out the operator latest release, we read this info from an environment variable that was set using `architect`.